src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setId(111L);
+        user.setLogin("reexxx");
+        user.setPassword("qwerty");
+        user.setAge(25);

```

Comment:
Create constants for these values

Reply:
I'm not ssure it's good idea. But will see, what mentor say about this